tsconfig.json

```diff
@@ -1,5 +1,5 @@
 {
-  "extends": "../../zlux-app-manager/virtual-desktop/plugin-config/tsconfig.base.json",
+  "extends": "./zlux/zlux-app-manager/virtual-desktop/plugin-config/tsconfig.base.json",
```

Comment:
@NakulManchanda please verify if this will cause any issues on existing build pipeline. Thanks!

Reply:
i think current pipeline will change after this, so should be fine